tmt/base.py

```diff
@@ -3899,7 +3899,7 @@ def _stop_running_guests(self, run: Run) -> bool:
                         self.cli_invocation.options['quiet'] = quiet
         return successful
 
-    def guests(self) -> bool:
+    def guests(self, id_: tuple[str, ...]) -> bool:
```

Comment:
How about giving the parameter a different name? After the patches that landed in this area I for one feel we have way too many "id" mentions around, and it's easy to forget what they mean. `id_` is very generic, how about `run_ids`? It would be immediately clear, and it would not collide with the built-in `id()` function.